libs/api/src/lib/media/karmaMediaAdapter.ts

```diff
@@ -96,7 +96,7 @@ export class KarmaMediaAdapter implements MediaAdapter {
   }
 
   async getImageURL(
-    {id, filename, extension, focalPoint}: ImageWithFocalPoint,
+    {id, filename, extension, focalPoint}: Image,
```

Comment:
Why did you remove her the focal point from the karma media Adapter getImageURL function?